docs/helm/README.md

```diff
@@ -65,6 +65,12 @@ the installation.
 
 For the best results we recommend [deploying ACS to AWS EKS](./eks-deployment.md). If you have a machine with at least 16GB of memory you can also [deploy using Docker for Desktop](./docker-desktop-deployment.md).
 
+The recommended cluster resources for the Enterprise version are at least 3
+nodes with 12 cpu cores and 32 GB of memory in total. You can install with lower
```

Comment:
Mentions that this is for the components enabled by default (which users may want to disable, or enable new ones)